src/Ouroboros/Network/Protocol/UntypedStream/Type.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE StandaloneDeriving #-}
+module Ouroboros.Network.Protocol.UntypedStream.Type where
```

Comment:
Misnomer? It looks like this module defines a typed stream.

Reply:
Yes and no :) - no because the types are ignored, I used `Protocol.Untyped.Peer` rather than `Protocol.Core.Peer`.  By adding one transition it would be possible to translate this to `Protocol.Core.Peer` if latency hiding worked out.